### DISCUSSION TITLE
KATA2 -  Entregables

### DISCUSSION THREAD

**Discussion by FernandoRizo** [LaunchX-InnovaccionVirtual/CursoIntroPython#34] [Q&A] [answered]
Buenas noches @FernandaOchoa tengo una duda acerca del producto entregable de la kata 2, hice todos los pasos pero al final no supe qué es lo que se debe de entregar de esta kata ya que solamente instalé bibliotecas y creé el entorno virtual en la consola (cmd). Agradecería bastante si pudieras contestar mi duda o algún otro compañero que si sepa :c

**Comment by DianaMagallanes** [comment, marked as answer]
Hola Fernando. 
Una captura de pantalla de la consola con los comandos.

**Reply by JesusYG** [reply]
![Imagen1](https://user-images.githubusercontent.com/99446340/154376790-978c6007-efd0-4e18-8d62-03f78d83eeda.png)
![Imagen22](https://user-images.githubusercontent.com/99446340/154377759-b10afd77-ce9f-4df5-b29c-d9923ac0d51f.png)

Las imagenes son de contexto.

Buen dia, tengo un error al final de la Kata 2,
 cuando intento ejecutar el comando: "pipdate" .
El error dice --->   'pipdate' is not recognized as an internal or external comman


@DianaMagallanes si pudieras ayudarme a entener el error te lo agradeseria mucho, como saber que realizando bien las cosas? Adicional, sabes hasta cuando es la fecha limite para regsitrar una cuenta github al programa actual LauchX github ?

**Comment by FernandoRizo** [comment]
Hola qué tal Jesús, intentaré ayudarte en lo que pueda. Veo que necesitas
abrir Visual Studio Code para ejecutar esa parte de código, en la consola
ya que estés en el los archivos donde tienes ese código intenta ejecutar
pipdate.
Según tengo entendido con el código es suficiente para ejecutarlo. (Adjunto
foto)

Tenemos hasta el lunes para entregar las katas por lo que antes del lunes
deberías de hacer la cuenta. Espero haberte ayudado.
Saludos.

**Comment by JesusYG** [comment]
@FernandoRizo muchas gracias lo voy a revisar, excelente noche!
